internal/gateways/publisher.go

```diff
@@ -452,6 +463,15 @@ func (p *publisher) updateIdentityStateTxStatus(ctx context.Context, state *doma
 	return nil
 }
 
+// groupByUserId - groups claims by user id
+func groupByUserId(claims []*domain.Claim) map[string][]string {
```

Comment:
if makes more sense to me returning a  map[string][]domain.Claims. Just cosmetic but in it explains better the purpose of the function

Reply:
If I return map[string][]domain.Claims I will have to iterate the claims later because the event expect for credentials ids (string). So, this cosmetic change generates an extra iteration process, not sure if we want that only for having a better or more explanatory function signature